### PR TITLE
Add age range prediction example

### PR DESCRIPTION
**Pull request by sakares**
This PR is about adding how to predict a person face age by deep learning architecture name `Vision Transformers`
We utilized the available Huggingface ViT [nateraw/vit-age-classifier](https://huggingface.co/nateraw/vit-age-classifier) model

ViT prediction code:

```python3
def get_age_range(img):
    
    # Transform our image and pass it through the model
    inputs = transforms(img, return_tensors='pt')
    outputs = model(**inputs)
    logits = outputs.logits
    predicted_class_idx = logits.argmax(-1).item()

    # Predicted Classes
    preds = model.config.id2label[predicted_class_idx]

    return preds

```
Example result: [link](https://github.com/sakares/ai-seed/blob/feature/face-age-range-classifier/Audio%20Visual/Problems/FaceAgeDetection_OpenCV.ipynb)

or screenshot here:

Screen Shot 2021-10-05 at 20 45 15

### CODE REVIEW

**Comment by sakares** [comment]
@Thilakraj1998 updated, please check
